apps/web/src/hooks/useMerkl.tsx

```diff
@@ -203,3 +203,27 @@ export default function useMerkl(poolAddress: string | null) {
     [claimTokenReward, hasMerkl, isTxPending, rewardsPerToken],
   )
 }
+
+export const useMerklApr = (lpAddress?: Address, chainId?: number, isMerklPool?: boolean) => {
```

Comment:
how about we combine this hook with `useMerklInfo`?

Reply:
sure